File: app/src/main/java/com/cn29/aac/datasource/itunes/db/ItunesDb.java
package com.cn29.aac.datasource.itunes.db;

import com.cn29.aac.repo.itunes.Album;
import com.cn29.aac.repo.itunes.Artist;

import androidx.room.Database;
import androidx.room.RoomDatabase;

/**
 * Created by Charles Ng on 19/10/2017.
 */
@Database(entities = {Artist.class, Album.class}, version = 3)
public abstract class ItunesDb extends RoomDatabase {

  public abstract ArtistDao artistDao();

  public abstract AlbumDao albumDao();
}

File: app/src/main/java/com/cn29/aac/datasource/itunes/db/ItunesDb.kt
package com.cn29.aac.datasource.itunes.db

import androidx.room.Database
import androidx.room.RoomDatabase
import com.cn29.aac.repo.itunes.Album
import com.cn29.aac.repo.itunes.Artist

@Database(entities = [Artist::class, Album::class], version = 3)
abstract class ItunesDb : RoomDatabase() {
    abstract fun artistDao(): ArtistDao
    abstract fun albumDao(): AlbumDao
}